### ISSUE TITLE
Doctor Profile Screen

### ISSUE THREAD

**Issue by HimeshNayak** [HimeshNayak/SwingerIOT#5]
Here are some issues generated from Axe-Core DevTools
```
[
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(12) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 4.06 (foreground color: #428d51, background color: #ffffff, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 15px; font-weight: normal; font-family: Poppins, Arial, sans-serif; color: rgb(66, 141, 81); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 16.8px; width: 78px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 318, 493); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:40:00.400Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(19) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 4.06 (foreground color: #428d51, background color: #ffffff, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1",
        "source": "<p style=\"text-align: left; line-height: 1.17188; font-size: 20px; font-weight: 500; font-family: Poppins, Arial, sans-serif; color: rgb(66, 141, 81); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 23.2px; width: 188px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 32, 539); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:40:00.400Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "impact": "serious",
        "isManual": false,
        "selector": [
            "div:nth-child(21) > div > p"
        ],
        "summary": "Fix any of the following:\n  Element has insufficient color contrast of 2.59 (foreground color: #ffffff, background color: #64b174, font size: 15.0pt (20px), font weight: bold). Expected contrast ratio of 3:1",
        "source": "<p style=\"text-align: center; line-height: 1.17188; font-size: 20px; font-weight: bold; font-family: Poppins, Arial, sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 23.2px; width: 281px; transform-origin: 0px 0px 0px; transform: matrix(1, 0, 0, 1, 67, 656); left: 0px; top: 0px;\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:40:00.400Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    },
    {
        "ruleId": "color-contrast",
        "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
        "help": "Elements must have sufficient color contrast",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome",
        "needsReview": true,
        "isManual": false,
        "selector": [
            "flt-picture > p"
        ],
        "summary": "Fix any of the following:\n  Element's background color could not be determined because it's partially obscured by another element",
        "source": "<p style=\"text-align: center; line-height: 1; font-size: 10px; font-weight: 900; font-family: sans-serif; color: rgb(255, 255, 255); position: absolute; white-space: pre-wrap; overflow-wrap: break-word; overflow: hidden; height: 9.6px; width: 80px; transform-origin: 0px 0px 0px; transform: matrix(0.707107, 0.707107, -0.707107, 0.707107, 364.81, -7.77817);\">",
        "tags": [
            "cat.color",
            "wcag2aa",
            "wcag143"
        ],
        "igt": "",
        "shareURL": "",
        "createdAt": "2021-08-01T06:40:00.400Z",
        "testUrl": "https://api.flutlab.io/res/projects/263970/jvjhcir0rt9nc36agxme/index.html?v=0.5287613885742697",
        "foundBy": "himeshnayak015@gmail.com"
    }
]
```
